src/components/ebay-legacy-floating-textbox/README.md

```diff
@@ -0,0 +1,31 @@
+# ebay-textbox
```

Comment:
Should we update these heading references to be `ebay-legacy-floating-textbox` ?

Reply:
Maybe a small little blurb about the purpose of this legacy component too.